bfdd/bfdd_vty.c

```diff
@@ -152,6 +156,8 @@ static void _display_peer(struct vty *vty, struct bfd_session *bs)
 		bs->timers.required_min_echo / 1000);
 
 	vty_out(vty, "\t\tRemote timers:\n");
+	vty_out(vty, "\t\t\tDetect-multiplier: %" PRIu32 "\n",
+		bs->remote_detect_mult);
 	vty_out(vty, "\t\t\tReceive interval: %" PRIu32 "ms\n",
```

Comment:
not under Remote timers.
Better to gather it with Detect Multiplier, and by changing the wording like for diagnostic:
```
Detect-multiplier
Remote detect-multiplier
```

Also, this change needs to be done for json too, please.

Reply:
Since multiplier value is closely related to timers, i thought it would be logical to club them them under local and remote timers.

Reply:
same comment as above